cds_dojson/marc21/fields/books/book.py

```diff
@@ -361,8 +371,7 @@ def external_system_identifiers(self, key, value):
             system_id.update({'value': sub_a,
                               'schema': sub_9})
         else:
-            # TODO check with CDS
-            raise ManualMigrationRequired
+            raise IgnoreKey('external_system_identifiers')
```

Comment:
actually, I think this should not be ignored. We should first do https://github.com/CERNDocumentServer/cds-migrator-kit/issues/15 which will clean our code of aleph number (number like `CERCER`), and then raise an exception if this is still present in the records. The reason is that these numbers are important for linking and we should make sure we correctly replace them by their record IDs.

Reply:
currently, these are the possible values in `035__9`:

`set(['CNUM-INSPIRE', 'IEEECONF', 'inspire', 'INSPIRE-CNUM', 'SCEM', 'arXiv', 'LHCLHC', 'SPIRES', 'DOE', '926408', 'ADMADM', 'Iinspire-CNUM', 'Inspire-CNUM', 'CERCER', 'INDICO.CERN.CH', 'SLACCONF', 'INIS', 'INPIRE-CNUM', 'DLC', 'HAL', 'DESY', 'FIZ', 'WAI01', 'http://inspirehep.net/oai2d', 'Isnpire-CNUM', 'INSPRIE-CNUM', 'SAFARI', 'Isnpire', 'SLAC', 'INSPIRE', 'AgendaMaker', 'KEK', 'Inspire', 'EBL', '290555', 'INSPEC', 'CERN annual report', 'CERN', '273873', 'inspire-CNUM'])` but it looks like only the `inspire-cnum/inspirecnum` are being treated in a slightly different way.
There are several things here that I think @agentilb could help with: 
    i) the normalization of these values, some of them are obviously typos (`Isnpire`).
    ii) it looks like `inspire` appears in various forms, do all relate to the same `inspire_cnum`?
    iii) the cleaning of these values, as there are also number there, that might be a mistake and they need to be in another subfield
    iv) only `CERCER` should potentially disappear (but only after doing https://github.com/CERNDocumentServer/cds-migrator-kit/issues/15), or there are others that could be ignored?

Reply:
After discussing with @kprzerwa we decided to close this PR as it is fulfilling the original request and start a new ticket https://github.com/CERNDocumentServer/cds-dojson/issues/167 to solve the issues with the data and to implement the changes